<launch>
    <!-- <node pkg="pcl_ros" type="pcd_to_pointcloud" name="pcd2pc" args="$(find static_map)/map_file/map_1010_cluster.pcd">
        <remap from="/cloud_pcd" to="/points_raw" />
        <param name="frame_id" value="map" />
    </node> -->
    <node pkg="static_map" type="static_map_node" name="static_map_node">
        <param name="pcd_file" value="$(find static_map)/map_file/map_1010_cluster.pcd" />
        <param name="map_frame" value="map" />
        <param name="duration" value="1.0" />
        <remap from="/static_map" to="/points_raw" />
    </node>
    <include file="$(find lidar_euclidean_cluster_detect)/launch/lidar_euclidean_cluster_detect.launch" />
</launch>